### ISSUE TITLE
1.18 Forge server fails to start with both Journeymap and Dynmap enabled.

### ISSUE THREAD

**Issue by BonaireDreams** [TeamJM/journeymap#358]
#### Instructions for Submitting an Issue
1. Ask for help on Discord (https://discord.gg/eP8gE69) #support if you're unsure this is a new issue.
2. Please search for existing issues before submitting a new one.
3. Please add the correct MC<version> label to your issue. Example: "[1.12.2] Waypoints are upside down"
4. Completely fill out the template below. Failure to do so is likely to get the issue closed immediately.
5. Prove you read these instructions by deleting this section before submitting. ;)
---
  
#### JourneyMap Version:
 1.18-5.8.0alpha8

#### Forge Version:
1.18-38.0.16

Latest Dynmap 3.3-snapshot for 1.18

#### Describe the problem in repeatable steps:

1. Server crashes with both Journeymap and Dynmap Installed
2. Seems both are trying to start the web engine.
3.

#### Relevant errors from `.minecraft/journeymap/journeymap.log`:
```
15:10:16
Exception in thread "main" java.lang.module.ResolutionException: Modules journeymap and dynmap export package org.eclipse.jetty.http to module supermartijn642configlib
at java.base/java.lang.module.Resolver.resolveFail(Resolver.java:901)
at java.base/java.lang.module.Resolver.failTwoSuppliers(Resolver.java:815)
at java.base/java.lang.module.Resolver.checkExportSuppliers(Resolver.java:736)
at java.base/java.lang.module.Resolver.finish(Resolver.java:380)
at java.base/java.lang.module.Configuration.<init>(Configuration.java:140)
at java.base/java.lang.module.Configuration.resolveAndBind(Configuration.java:494)
at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.ModuleLayerHandler.buildLayer(ModuleLayerHandler.java:70)
at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:61)
at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.Launcher.run(Launcher.java:104)
at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
at MC-BOOTSTRAP/cpw.mods.modlauncher@9.0.7/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
at cpw.mods.bootstraplauncher@0.1.17/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:90)

```

**Comment by BonaireDreams** [comment]
Some info Mikeprimm (dynmap Author) found with how Forge is handling the modules. Hoping it gives you some info as well.
`This is actually a Forge 1.18-ism - it appears they are being clever around module dependencies, and parsing the META-INF files in the mods. Both dynmap and journeymap include jetty in their shaded uberjars, but dynmap rebases the packages - there are literally no org.eclipse.jetty classes there, but Forge (as of 1.18 - this situation with Jetty files is quite old for both mods...) is apparently tripping over the junk META-INF/maven files that were not stripped when the uberjars were built. I'm updating the dynmap build to strip these files, as I'm sure this will be easier to do than to get forge do behave better on this point.`

**Comment by mysticdrew** [comment]
https://github.com/webbukkit/dynmap/issues/3556

I am going to strip them on my end also, thanks for giving me this info.

**Comment by mysticdrew** [comment]
Fixed in 5.8.0 Alpha9 for 1.18 and beta3 for 1.16.5 and 1.17.1

**Comment by BonaireDreams** [comment]
Confirmed Tested and working. Updated server with latest Dynmap build and JourneyMap Alpha9, Server starts fine and everything is functioning. Reporting this issue closed.
